proposals/fp_interop/25-xxx.txt

```diff
@@ -0,0 +1,170 @@
+To: J3                                                     J3/##-###
+From: Patrick Fasano
+Subject: Requirements and specifications for interoperability with C
+         interchange and extended types
+Date: 2025-Oct-XX
+
+Reference:
+
+1 Background
+============
+
+bg01. The ISO C23 specification (ISO/IEC 9899:2024) added support for
+      interchange and extended floating-point types. The current Fortran
+      202Y work list WG5/N2249 includes adding interoperability with
+      these types as accepted work item US05. At meeting #230 in June
+      2023, J3 accepted paper 23-176 as background/use-case for
+      additional interchange types.
+
+bg03. This paper contains formal requirements and specifications for
+      Fortran 202Y proposal US05, "Add extended floating-point types
+      defined in ISO C23 to ISO_C_BINDING".
+
+2 Requirements
+==============
+
```

Comment:
Suggest an introductory section explaining that many of these are optional (as marked) and can be independently dropped while still keeping the rest of the paper